vfio/src/vfio_device.rs

```diff
@@ -242,41 +239,7 @@ impl VfioGroup {
 
         container.set_iommu(VFIO_TYPE1v2_IOMMU)?;
 
-        Self::kvm_device_add_group(&device, &group)?;
```

Comment:
I think you just need to not add the group to the KVM device for "fixing" that issue.